pkg/search/paginated/paginated.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/stackrox/rox/pkg/search"
 )
 
+const Unlimited = 0
```

Comment:
IDK, Do you think maybe it would be better to pick a realistic max for a page instead and use that?  I think that would feel cleaner and more maintainable.

Reply:
I find setting a default page size is a good idea.
However, this goes beyond the scope of this refactor, IMHO.

Reply:
> realistic max

How this will work then?